src/doc/trpl/compound-data-types.md

```diff
@@ -196,8 +196,8 @@ Now, we have actual names, rather than positions. Good names are important,
 and with a struct, we have actual names.
 
 There _is_ one case when a tuple struct is very useful, though, and that's a
-tuple struct with only one element. We call this a *newtype*, because it lets
-you create a new type that's similar to another one:
+tuple struct with only one element. We call this the *newtype* pattern, because
+it lets you create a new type that adds semantic meaning to another one:
```

Comment:
It's not just additive, though, as newtypes don't inherit all of the behavior of the old one, it's an entirely new thing.